app/src/androidTest/java/com/google/maps/android/compose/GoogleMapViewTests.kt

```diff
@@ -281,6 +282,30 @@ class GoogleMapViewTests {
             .performClick()
     }
 
+    @Test
```

Comment:
Does this test actually pass?

Reply:
yes it is.

When I run the test with the existing `rememberMarkerState`, the test fails as below.
<img width="825" alt="스크린샷 2024-12-16 23 14 02" src="https://github.com/user-attachments/assets/af46bf75-4a30-443c-836b-983ab60425f1" />

When I run the test with `rememberUpdatedMarkerState`, the test succeeds as shown below.
<img width="817" alt="스크린샷 2024-12-16 23 15 52" src="https://github.com/user-attachments/assets/6c8fe273-771c-4b95-8575-95e1a2eb0bbd" />

I can't verify this because there is no CI in this repo, but I think you can verify it by running the tests locally.